src/main/java/org/jahia/modules/jahiaoauth/connectors/FranceConnectConnectorImpl.java

```diff
@@ -24,49 +24,22 @@
 package org.jahia.modules.jahiaoauth.connectors;
 
 import org.jahia.modules.jahiaauth.service.ConnectorConfig;
-import org.jahia.modules.jahiaauth.service.ConnectorPropertyInfo;
 import org.jahia.modules.jahiaoauth.service.OAuthConnectorService;
 
-import java.io.IOException;
-import java.util.List;
 import java.util.Map;
 
-public class FranceConnectConnectorImpl implements OAuthConnectorService {
+public class FranceConnectConnectorImpl extends Connector implements OAuthConnectorService {
 
     private Map<String, String> protectedResourceUrl;
```

Comment:
This field is shading the `protectedResourceUrl` field from Connector class with different type and visibility. You should use another name.